src/component/person.jsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable max-len */
+import React from 'react';
+import './person.scss';
+
+const Married = (isMarried, sex, partnerName) => {
```

Comment:
```suggestion
const married = (isMarried, sex, partnerName) => {
```

Reply:
Married it's not the component, it's just separate function
Will be better to use better naming, like getMarriedStatus()